### PR TITLE
Add a build for an extended utility client

### PR DESCRIPTION
**Pull request by tmckayus**
The spark application launch scripts need some additional CLI
commands to build and configure a cluster/dirver, but don't need
to include the entire "oc" client. This change adds a build for an
extended oshinko CLI via golang build constraints.

### CODE REVIEW

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -64,8 +50,8 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	}
 
 	templates.ActsAsRootCommand(cmds, filters, groups...).
-		ExposeFlags(getCmd, "server", "client-certificate",
-			"client-key", "certificate-authority", "insecure-skip-tls-verify", "token")
+		ExposeFlags(groups[0].Commands[0], "server", "client-certificate",
```
Comment (crobby) [inline pkg/cmd/cli/cli.go]: I'm mostly unfamiliar, but is there any chance that the magic number 0's ever become not-so-magic?

Comment (tmckayus) [inline pkg/cmd/cli/cli.go]: I don't think so. This is just like a Python index into a list.  The CommandGroups object is a list of CommandGroup objects, which contain a list of Commands.

Hmm, there is another way .... mutiple return values.  Good idea!

Comment (tmckayus) [inline pkg/cmd/cli/cli.go]: abstraction preserved :)

File: pkg/cmd/cli/cli.go [hunk 2]
```diff
@@ -43,19 +41,7 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	}
 
 	f := clientcmd.New(cmds.PersistentFlags())
-
-	getCmd := oshinkocmd.NewCmdGet(fullName, f, in, out)
```
Comment (sub-mod) [inline pkg/cmd/cli/cli.go]: This is one anchor point I was talking about.
It is ok to push to a method

**Comment by sub-mod** [comment]
LGTM

**Pull request merged**